src/integrations/feed/generators.ts

```diff
@@ -123,11 +124,14 @@ export const feedGenerators: Partial<Record<FeedVersion, FeedGenerator>> =
     ),
   });
 
-export const versionToFeedGenerator = (version: number): FeedGenerator => {
+export const versionToFeedGenerator = (
+  version: number | FeedVersion,
+  inputOpts?: Options,
+): FeedGenerator => {
   return new FeedGenerator(
     feedClient,
     new FeedLofnConfigGenerator(baseFeedConfig, lofnClient, {
-      ...opts,
+      ...(inputOpts ?? opts),
```

Comment:
I think we should just instantiate lofn generator in `schema/feed` as we do for other non main feeds, this passing of opts multiple times is complex as is.

Reply:
Should not be the issue, just copy paste to schema right?